solutions/cses-1619.mdx

```diff
@@ -184,6 +188,72 @@ int main(){
 }
 ```
 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java solutions using this implementation will TLE on test case #6.
+</Warning>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class RestaurantCustomers {
+	static class Customer {
+		int time;
+		// a type of 1 means that it is an arrival
+		// and a type of 2 means that it is a departure
```

Comment:
Should't a type of -1 mean that it's a departure, seeing as you set the type to -1 for the ends of the intervals?

Reply:
yep a mistake